website/docs/foundations/icons/index.js

```diff
@@ -25,14 +21,18 @@ export default class Index extends Component {
   });
 
   get filteredIcons() {
-    if (this.searchQuery) {
+    // this.router.currentRoute.queryParams['searchQuery'];
+    const searchQuery = this.router.currentRoute.queryParams['searchQuery'];
```

Comment:
I would add a comment explaining that these queryParams are to be found also in the `show.js` file, why they're are there, and recommend to keep the names in sync

Reply:
added a comment